Running git on the source tarball cannot return useful information, so just disable it.

Author: Alexander Grund (TU Dresden)

diff --git a/setup.py b/setup.py
--- a/python/setup.py
+++ b/python/setup.py
@@ -739,6 +739,7 @@ def get_git_branch():
 
 
 def get_git_version_suffix():
+    return ''
     branch = get_git_branch()
     if branch.startswith("release"):
         return ""
